src/games/controller.ts

```diff
@@ -87,21 +82,36 @@ export default class GameController {
     const game = await Game.findOneById(id)
     if (!game) throw new BadRequestError(`Game does not exist`)
 
+    const toSend=gameToSend(game)
     const player = await Player.findOne({ user, game })
     if (!player)
```

Comment:
It's checking if the user is a player for the game, if not we send the game but with both boards transformed into guessBoards, this will help us to allow other users to watch a game and just seeing the guessBoards not the boards with details of ships

Reply:
ohhhh ok. I was still thinking in only players are seeing the game so then it didn't make much sense to me XD so we can have watchers now haha whyyyyy but ok why not XD

Reply:
Haha, just following the starter style :stuck_out_tongue: . 
maybe we will organize a tournament later, then people can watch :joy: